### PR TITLE
In progress -Employee active ui

### PR DESCRIPTION
**Pull request by surekhag**

### CODE REVIEW

File: client/src/api/employeeApi.js [hunk 1]
```diff
@@ -2,16 +2,24 @@ import axios from 'axios'
...
 export function deleteEmployeeApi(id) {
-  return axios.delete(DELETE_USER_URL + id)
+  return axios.delete(USER_URL + id)
+}
+export function loadActiveEmployeeApi(data) {
+ 
+  const query = {
+    status:{data}
+  }
+   console.log("in api", data)
```
Comment (bhushan-oe) [inline client/src/api/employeeApi.js:23]: remove console

File: client/src/sagas/watchers/employeeInfo.js [hunk 2]
```diff
@@ -10,32 +10,51 @@ import {
...
-function* workerEmployeeInfoSaga() {
+function* workerEmployeeInfoSaga({ payload }) {
+  const { data } = payload
   try {
-    const employees = yield call(loadAllEmployeeData)
-    yield put(setAllEmployeeData(employees))
-  } catch (e) { }
+    if (data) {
+      const employees = yield call(loadActiveEmployeeApi, data)
+      console.log("active",employees)
```
Comment (bhushan-oe) [inline client/src/sagas/watchers/employeeInfo.js:29]: remove console

File: client/src/sagas/watchers/employeeInfo.js [hunk 3]
```diff
@@ -10,32 +10,51 @@ import {
...
+      yield put(setActiveEmployeeData(employees))
+    } else {
+      const employees = yield call(loadAllEmployeeData)
+       console.log('all',employees)
```
Comment (bhushan-oe) [inline client/src/sagas/watchers/employeeInfo.js:33]: remove console

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

**Comment by bhushan-oe** [comment]
run eslint and format. resolve conflicts

**Comment by surekhag** [comment]
We donr need this code anymore ..

**Pull request closed without merging**